generated/setup.py

```diff
@@ -32,7 +32,7 @@
     author_email="support@groundlight.ai",
     url="",
     keywords=["OpenAPI", "OpenAPI-Generator", "Groundlight API"],
-    python_requires=">=3.6",
+    python_requires=">=3.9",
```

Comment:
I think this is the only change I didnt capture - although this file is generated so I'm not sure if its the right place to make the change.

Reply:
Yeah, this file AFAIK isn't actually used by anything.